### DISCUSSION TITLE
Batched PUT requests to InfluxDB for performance

### DISCUSSION THREAD

**Discussion by robagar** [eclipse-zenoh/roadmap#155] [Ideas and new features]
We've found that writing fairly high frequency data to InfluxDB 1.8 via the zenoh-influxdb-backend plugin can fall behind from making individual PUT requests for each data point.  

The InfluxDB docs recommend [sending data in batches](https://docs.influxdata.com/influxdb/v1/guides/write_data/#writing-multiple-points) for best performance

Our implementation is just in a fork here - 

https://github.com/engineeredarts/zenoh-backend-influxdb/tree/TR-7332-zenoh-0.11-influxdb

But it would be awesome to add something like this to the official plugin.  Should I make a PR?  Or maybe wait for the upcoming 1.0 release?

**Comment by gabrik** [comment]
Hi @robagar,

We are open to accepting PRs, so please feel free to open one against https://github.com/eclipse-zenoh/zenoh-backend-influxdb.

We are in the process of release so it would be nice if this patch could land before the final release.

**Reply by robagar** [reply]
Thanks - https://github.com/eclipse-zenoh/zenoh-backend-influxdb/pull/205

Draft for now as not yet tested - there seems to be [a problem loading the storage plugin 1.0.0-beta.1 ](https://discordapp.com/channels/914168414178779197/940584045287460885/1280453346280472639)
